scripts/commands/updateApiDocs.ts

```diff
@@ -100,10 +98,13 @@ zxMain(async () => {
   );
 
   const artifactFolder = pkg.ciArtifactFolder();
```

Comment:
The artifactFolder's verbosity annoys me - it's such a long path. But probably best for us to change in a follow up, since it's not strictly necessary here.